File: DataSummitSolution/DataSummitModels/DTO/PropertyData.cs
using System;
using System.Collections.Generic;
using System.Text;

namespace DataSummitModels.DTO
{
    public class PropertyData
    {
        //From ProfileAttribute class
        public long ProfileAttributeId { get; set; }
        public string Name { get; set; }
        public int NameX { get; set; }
        public int NameY { get; set; }
        public short NameWidth { get; set; }
        public short NameHeight { get; set; }
        public byte PaperSizeId { get; set; }
        public byte BlockPositionId { get; set; }
        public int? ProfileVersionId { get; set; }
        public DateTime? CreatedDate { get; set; }
        public long? UserId { get; set; }
        public string Value { get; set; }
        public int? ValueX { get; set; }
        public int? ValueY { get; set; }
        public short? ValueWidth { get; set; }
        public short? ValueHeight { get; set; }
        
        //From StandardAttribute class
        public short StandardAttributeId { get; set; }
        public string StandardName { get; set; }

        //From Sentence class
        public Guid SentenceId { get; set; }
        public string Words { get; set; }
        public int Width { get; set; }
        public int Height { get; set; }
        public int Left { get; set; }
        public int Top { get; set; }
        public string Vendor { get; set; }
        public bool IsUsed { get; set; }
        public decimal? Confidence { get; set; }
        public decimal? SlendernessRatio { get; set; }
        public long DrawingId { get; set; }

        //From Properties class
        public long PropertyId { get; set; }

        public PropertyData()
        { }
    }
}

File: DataSummitSolution/DataSummitUI/src/app/properties/models/propertyData.ts

export class PropertyData {

    ProfileAttributeId: number;
    Name: string;
    NameX: number;
    NameY: number;
    NameWidth: number;
    NameHeight: number;
    PaperSizeId: number;
    BlockPositionId: number;
    ProfileVersionId: number;
    CreatedDate: Date;
    UserId: number;
    Value: string;
    ValueX: number;
    ValueY: number;
    ValueWidth: number;
    ValueHeight: number;
    StandardAttributeId: number;
    StandardName: string;
    SentenceId: string;
    Words: string;
    Width: number;
    Height: number;
    Left: number;
    Top: number;
    Vendor: string;
    IsUsed: Boolean;
    Confidence: number;
    SlendernessRatio: number;
    DrawingId: number;
    PropertyId: number;

    constructor(profileAttributeId?: number,
        name?: string, nameX?: number, nameY?: number, nameWidth?: number, nameHeight?: number,
        paperSizeId?: number, blockPositionId?: number, profileVersionId?: number,
        createdDate?: Date, userId?: number,
        value?: string, valueX?: number, valueY?: number, valueWidth?: number, valueHeight?: number,
        standardAttributeId?: number, standardName?: string, sentenceId?: string,
        words?: string, width?: number, height?: number, left?: number, top?: number,
        vendor?: string, isUsed?: Boolean, confidence?: number, slendernessRatio?: number, drawingId?: number,
        propertyId?: number)
    {
        this.ProfileAttributeId = profileAttributeId || null; 
        this.Name = name || null; 
        this.NameX = nameX || null; 
        this.NameY = nameY || null; 
        this.NameWidth = nameWidth || null; 
        this.NameHeight = nameHeight || null; 
        this.PaperSizeId = paperSizeId || null; 
        this.BlockPositionId = blockPositionId || null; 
        this.ProfileVersionId = profileVersionId || null; 
        this.CreatedDate = createdDate || null; 
        this.UserId = userId || null; 
        this.Value = value || null; 
        this.ValueX = valueX || null; 
        this.ValueY = valueY || null; 
        this.ValueWidth = valueWidth || null; 
        this.ValueHeight = valueHeight || null; 
        this.StandardAttributeId = standardAttributeId || null; 
        this.StandardName = standardName || null; 
        this.SentenceId = sentenceId || null; 
        this.Words = words || null; 
        this.Width = width || null; 
        this.Height = height || null; 
        this.Left = left || null; 
        this.Top = top || null; 
        this.Vendor = vendor || null; 
        this.IsUsed = isUsed || null; 
        this.Confidence = confidence || null; 
        this.SlendernessRatio = slendernessRatio || null; 
        this.DrawingId = drawingId || null; 
        this.PropertyId = propertyId || null;
    }    
}